taskiq_redis/redis_broker.py

```diff
@@ -123,8 +123,12 @@ async def listen(self) -> AsyncGenerator[bytes, None]:
         :yields: broker messages.
         """
         redis_brpop_data_position = 1
-        async with Redis(connection_pool=self.connection_pool) as redis_conn:
-            while True:
-                yield (await redis_conn.brpop(self.queue_name))[
-                    redis_brpop_data_position
-                ]
+        while True:
+            try:
+                async with Redis(connection_pool=self.connection_pool) as redis_conn:
```

Comment:
This will be very inefficent, because it opens connection every time it tries to receive a message. To implement this functionality it's better to create a second while loop that wraps original one.

Reply:
From my understanding, the Redis ConnectionPool creates connections lazily and does not destroy them by default. This means that the same connection will be used unless it breaks or another client from the same pool creates an additional connection, leading to round-robin distribution between clients. In summary, the behavior appears to be consistent.

Reply:
@s3rius please let me know what do you think?